pages/workspace/tracking-plan/importing.mdx

```diff
@@ -6,49 +6,177 @@ title: 'Importing'
 
 import Link from '../../../components/Link';
 
-**This feature is currently in beta. Reach out to us to get early access.** 
+**This feature is currently in beta. Reach out to us to get early access.**
 
 Import your existing tracking plan into Avo to get started quickly. The Importer supports multiple data formats and allows you to review and choose which events and properties you'd like to import.
 
 <img
-    src={require('../../../images/avo-importer.png')}
-    width="100%"
-    alt="Screenshot of the Avo Importer"
-  />
+  src={require('../../../images/workspace/importing/avo-importer.png')}
+  width="100%"
+  alt="Screenshot of the Avo Importer"
+/>
 
 ## Importing from Amplitude
 
-To export from Amplitude open up Govern (at the bottom of the left sidebar) and then click the download arrow:
+1. Export your Tracking Plan from Amplitude. Open up Govern (at the bottom of the left sidebar) and then click the download arrow:
 
 <img
-    src={require('../../../images/govern-download.png')}
-    width="100%"
-    alt="Screenshot of Amplitude Govern"
-  />
+  src={require('../../../images/workspace/importing/govern-download.png')}
+  width="100%"
+  alt="Screenshot of Amplitude Govern"
+/>
 
-In the popup, select "Schema of all events and their properties" and click download:
+2. In the popup, select "Schema of all events and their properties" and click download:
 
 <img
-    src={require('../../../images/govern-download-select-schema.png')}
-    width="100%"
-    alt="Screenshot of the download modal on Amplitude Govern"
-  />
+  src={require('../../../images/workspace/importing/govern-download-select-schema.png')}
+  width="100%"
+  alt="Screenshot of the download modal on Amplitude Govern"
+/>
 
-Finally open the Importer from your Avo workspace and drag the downloaded .csv file to the drop zone or click it and select the file:
+3. Finally open the Importer from your Avo workspace and drag the downloaded .csv file to the drop zone or click it and select the file:
 
+<img
+  src={require('../../../images/workspace/importing/importer-select.png')}
+  width="100%"
+  alt="Screenshot of the Avo Importer selecting files"
+/>
+
+4. Review your events and finish the import 🎉
+
+## Importing from Mixpanel
+
+1. Export your Tracking Plan from Mixpanel. Open up Lexicon (in the top right corner click Data Management > Lexicon) and then click the export button:
```

Comment:
As this section is about exporting from Mixpanel it seems redundant to rephrase it the first step.
```suggestion
1. Open up Lexicon (in the top right corner click Data Management > Lexicon) and then click the export button:
```

Reply:
Agreed! shipping